manager.go

```diff
@@ -182,20 +174,21 @@ func (m *Manager) GetStats() (Stats, error) {
 			})
 		}
 		stats.Jobs[ns+queue] = jobs
-		qLen[ns+queue] = pipe.LLen(fmt.Sprintf("%squeue:%s", ns, queue))
+		q = append(q, queue)
 	}
 
-	_, err := pipe.Exec()
+	storeStats, err := m.opts.store.GetAllStats(q)
 
 	if err != nil && err != redis.Nil {
```

Comment:
should `redis.Nil` here be `storage.NoMessage`?

Reply:
Thanks for spotting this; the second check should be removed, `redis.Nil` does not make it the calling code.